Update the SVG Icons article (#1589)
DevExpress/devextreme-documentation: concepts/60 Themes and Styles/30 Icons/60 SVG Icons.md

> In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme</a> repository on GitHub.

Reviewer: ```suggestion
In addition to font icons, DevExtreme supplies the same icons in the SVG format. You can find SVG icons in the <a href="https://github.com/DevExpress/DevExtreme/tree/20_2/images/icons" target="_blank">DevExtreme repository</a> on GitHub.
```

> The following code demonstrates how to use SVG icons in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used with any other UI component that has the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') property. 

Reviewer 2: ```suggestion
The following code uses SVG icons in the [Button](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Button/Icons) widget. The same technique can be used with any other UI component that has the [icon](/api-reference/10%20UI%20Widgets/dxButton/1%20Configuration/icon.md '/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#icon') property. 
```

1. **Use the icon's URL**:

        <!-- tab: JavaScript -->
        new DevExpress.ui.dxButton(targetElement, {
            icon: "https://path/to/the/icon.svg"
        });

1. **Insert SVG content inline**:

        <!-- tab: JavaScript -->
        new DevExpress.ui.dxButton(targetElement, {
            icon: "<svg>SVG_CONTENT</svg>"
        });

1. **Import the icon**:

        <!-- tab: JavaScript -->
        import * as myIcon from "./assets/icon.svg";
        new DevExpress.ui.dxButton(targetElement, {
            icon: myIcon
        });

> [important]The SVG format allows running executable code that might be malicious. We strongly recommend that you use SVG icons only from trusted sources.

Reviewer 2: 1) `The SVG format allows running executable code that might be malicious.`

The original sentence specified _who_ ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.
Can we go back to the original sentence ("allows you to run executable code")?

2) `We strongly recommend that you use SVG icons only from trusted sources.`

This sentence is fine.

Author: > The original sentence specified who ("you") could potentially run malicious code. By changing it to the -ing form, we lose this.

That was the motivation behind this change. It is not a common practice to put any executable code in SVG. So, if an SVG icon contains code, is is most likely malicious, and the reader never wants to run it.

Reviewer 2: Thanks for the clarification. I understand now. Can we say the following?

`The SVG format could allow potentially malicious code to run. We strongly recommend that you use SVG icons only from trusted sources.`
